Autopilot/Inc/ultrasonic.hpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * Ultrasonic Sensor Functions
+ * Author: Dhruv Upadhyay, February 2021
+ */
+
+ #ifndef ULTRASONIC_HPP
+ #define ULTRASONIC_HPP
+
+ #include <stdint.h>
+
+
+const int SOUND_SPEED = 343; // In meters per second, used to calculate distance based on time taken to send and receive ping
```

Comment:
Can this be made static inside the cpp file?

Reply:
sounds good